app/models/loan.rb

```diff
@@ -59,6 +57,16 @@ def self.txn_mode_choices
     TXN_MODES
   end
 
+  # These methods should go away soon.
```

Comment:
i am looking fwd to them going away; is this a recommendation for work after this push on argentina migration?